content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -16,173 +16,69 @@ This feature requires Kubernetes version >= 1.18.
+Cert-manager has added [experimental Support for Kubernetes `CertificateSigningRequests`](https://cert-manager.io/docs/usage/kube-csr/) starting from version 1.4
 
-## Part 1: Using Kubernetes CA
+## Deploy Custom CA controller in the Kubernetes cluster
 
-{{< warning >}}
-Note that this example should only be used for basic evaluation. The use of the `kubernetes.io/legacy-unknown` signer is NOT recommended in production environments.
-{{< /warning >}}
+1. Deploy cert-manager according to the [installation doc](https://cert-manager.io/docs/installation/).
+   {{< warning >}}
+   Note: Make sure to enable feature gate: `--feature-gates=ExperimentalCertificateSigningRequestControllers=true`
+   {{< /warning >}}
 
-### Deploying Istio with Kubernetes CA
+1. Create three self signed cluster issuers `istio-system`, `foo` and `bar` for cert-manager.
+   Note: Namespace issuers and other types of issuers can also be used.
 
-1. Deploy Istio on the cluster using `istioctl` with the following configuration.
+## Export root certificate for cluster issuer
 
     {{< text bash >}}
-    $ cat <<EOF > ./istio.yaml
-      apiVersion: install.istio.io/v1alpha1
-      kind: IstioOperator
-      spec:
-        components:
-          pilot:
-            k8s:
-              env:
-              # Indicate to Istiod that we use a Custom Certificate Authority
-              - name: EXTERNAL_CA
-                value: ISTIOD_RA_KUBERNETES_API
-              # Tells Istiod to use the Kubernetes legacy CA Signer
-              - name: K8S_SIGNER
-                value: kubernetes.io/legacy-unknown
-      EOF
-    $ istioctl install --set profile=demo -f ./istio.yaml
-    {{< /text >}}
-
-1. Deploy the `bookinfo` sample application in the bookinfo namespace.
-    Ensure that the following commands are executed in the Istio root directory.
-
-    {{< text bash >}}
-    $ kubectl create ns bookinfo
-    $ kubectl apply -f <(istioctl kube-inject -f samples/bookinfo/platform/kube/bookinfo.yaml) -n bookinfo
-    {{< /text >}}
-
-### Verify that the certificates installed are correct
-
-When the workloads are deployed, above, they send CSR Requests to Istiod which forwards them to the Kubernetes CA for signing.
-If all goes well, the signed certificates are sent back to the workloads where they are then installed.
-To verify that they have been signed by the Kubernetes CA, you need to first extract the signed certificates.
-
-1. Dump all pods running in the namespace.
-
-    {{< text bash >}}
-    $ kubectl get pods -n bookinfo
-    {{< /text >}}
-
-    Pick any one of the running pods for the next step.
-
-1. Get the certificate chain and CA root certificate used by the Istio proxies for mTLS.
-
-    {{< text bash >}}
-    $ istioctl pc secret <pod-name> -o json > proxy_secret
-    {{< /text >}}
+    $ export istioca=$(kubectl get clusterissuers istio-system -o jsonpath='{.spec.ca.secretName}' | xargs kubectl get secret -n cert-manager -o jsonpath='{.data.ca\.crt}' |base64 -d)
```

Comment:
I do notice that there is not space between the pipe and base64 while we do have spaces around the other pipes. Not sure if this really matters and is only a nit.

Reply:
this does not matter :)